src/esp/assets/ResourceManager.cpp

```diff
@@ -1583,7 +1598,21 @@ void ResourceManager::addComponent(
           std::to_string(metaData.materialIndex.first + materialIDLocal);
     }
 
-    createGenericDrawable(mesh, node, lightSetupKey, materialKey, drawables);
+    gfx::Drawable::Flags meshAttributeFlags{};
+    if (meshes_[meshID]->getMeshData()->hasAttribute(
```

Comment:
These aren't failing but should either do the unwrap conditionally or check with an assert.

Reply:
Yes, will do.